blackbox/docs/configuration.txt

```diff
@@ -701,6 +701,18 @@ Collecting Stats
   expressive results but also in more occupied RAM. Setting it to
   ``0`` disables collecting job information.
 
+.. _stats.jobs_log_expiration:
+
+**stats.jobs_log_expiration**
+  | *Default:*  ``5s``
+  | *Runtime:*  ``yes``
+
+  This setting specifies the duration how long jobs kept in the ``sys.jobs_log``
```

Comment:
how about this phrasing ? 

This setting specifies the period during which the jobs are kept in the ``sys.jobs_log`` table.